### PR TITLE
Crowd Strike malware content

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 1]
```diff
@@ -375,7 +375,7 @@ script:
       secret: false
     - default: false
       description: Optional comment to add to the detection. Comments are displayed
-        with the detection in Falcon and are usually used to provide context or notes
+        with the detection in CrowdStrike Falcon and provide context or notes
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:378]: the description should be in "....."

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 2]
```diff
@@ -420,7 +420,7 @@ script:
   - arguments:
     - default: false
       description: The host agent ID (AID) of the host you want to contain. Get an
-        agent ID from a detection
+        agent ID from a detection.
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:423]: "...."

File: Packs/CrowdStrikeFalcon/README.md [hunk 3]
```diff
@@ -7,9 +7,19 @@ This SaaS platform provides a holistic solution for protecting enterprise assets
 - Provides real-time response features
 
 ## Playbooks
-This content pack includes the following playbooks. These playbooks auto isolate/unisolate endpoints by the device ID that was provided in the playbook. The playbooks can be used as part of a Malware investigation.
```
Comment (David-BMS) [inline Packs/CrowdStrikeFalcon/README.md:10]: @julieschwartz18 Please review this file as well

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 4]
```diff
@@ -1345,8 +1345,8 @@ script:
       required: false
       secret: false
     - default: false
-      description: The number of days for which the indicator should be valid. This
-        only applies to domain, ipv4, and ipv6 types. Default is 30.
+      description: The number of days the indicator should be valid for. This
+        only applies to domain, ipv4, and ipv6 types.
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:1349]: ".."

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 5]
```diff
@@ -1452,8 +1452,8 @@ script:
...
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:1456]: "...."

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 6]
```diff
@@ -3098,22 +3098,22 @@ script:
...
       isArray: false
       name: file_path
       required: true
       secret: false
     - auto: PREDEFINED
       default: false
-      description: The operatin system of the hosts given. As the revome command is
-        different in each operatin system, you can choose only one operating system.
+      description: The operating system of the hosts given. Since the remove command is
+        different in each operating system, you can choose only one operating system.
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:3116]: "..."

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Get_Detections_by_Incident_README.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+## Playbook Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| CrowdStrike.Detection.Behavior | CrowdStrike Detection Details. | string |
+
+## Playbook Image
+---
+![Crowdstrike Malware - Incident Enrichment](../doc_files/CrowdStrike_Falcon_-_Incident_Enrichment.png)
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Get_Detections_by_Incident_README.md:35]: is it the right image?

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Search_Endpoints_By_Hash.yml [hunk 8]
```diff
@@ -0,0 +1,258 @@
+id: CrowdStrike Falcon - Search Endpoints By Hash
+version: -1
+fromversion: 6.5.0
+name: CrowdStrike Falcon - Search Endpoints By Hash
+description: "This playbook searches across the organization for other endpoints\
+  \ associated with a specific SHA256 hash. \n"
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Search_Endpoints_By_Hash.yml:6]: we can remove the last \n

File: Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_True_Positive_Incident_Handling.yml [hunk 9]
```diff
@@ -0,0 +1,1318 @@
...
+- key: OperatingSystemToRemoveFrom
+  value: {}
+  required: false
+  description: |-
+    Values can be
+    Windows, Linux, Mac
+  playbookInputQuery:
+outputs: []
+tests:
+- Test - CrowdStrike Falcon
```
Comment (ssokolovich) [inline Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_True_Positive_Incident_Handling.yml:1318]: test?

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**